Docker topic for Docs

agnar-nomad opened the pull request:
In this PR, I am adding a `.md` file describing how to Dockerise a waku app. This short article is supposed to be a part of future documentation.

dai-shi commented on docs/guides/docker.md:
```diff
@@ -0,0 +1,80 @@
+Bundle your Waku app into a Docker container for portability and easy deployment in various environments.
+
+### Prerequisites
+Make sure you have Docker and Node.js installed on your machine. I used `yarn` as my packager manager, customise setup to your manager. We will prepare the image and container in 3 easy steps for reproducible results. Create 3 files in your folder root: `Dockerfile`, `.dockerignore`, `docker-compose.yml`.
```
> I used `yarn` as my packager manager

It's a weird phrase as a guide. Something like "In this guide, we use `yarn`..."

sandren replied:
It should be changed to use `npm` instead for consistency.

agnar-nomad replied:
The sentence has been removed.
All mentions of `yarn` were replaced by `npm`

dai-shi reviewed (commented):
@sandren would you give any advice on writing guides?

sandren commented:
@dai-shi It looks good. I would just suggest that we standardize the format to MDX with the following intro:

```mdx
---
slug: guide/[slug]
title: Title
description: One sentence description.
---

# Title
```

agnar-nomad commented:
I incorporated all all the above notes.
I added the mdx header as requested, however the subsequent article title is put as an H2, for consistency with the other mdx files in the repo.

dai-shi reviewed (commented):
Thanks for the update. Please run prettier to format docs.

agnar-nomad commented:
Should be formatted correctly now

dai-shi reviewed (approved):
Thanks for your contribution!
